Tests/Marketplace/upload_packs.py

```diff
@@ -471,6 +474,52 @@ def get_recent_commits_data(content_repo):
     return content_repo.head.commit.hexsha, content_repo.commit('origin/master~1').hexsha
 
 
+def check_if_index_is_updated(index_folder_path, content_repo, current_commit_hash, remote_previous_commit_hash):
+    """ Checks stored at index.json commit hash and compares it to current commit hash. In case no packs folders were
+    added/modified/deleted, all other steps are not performed.
+
+    Args:
+        index_folder_path (str): index folder full path.
+        content_repo (git.repo.base.Repo): content repo object.
+        current_commit_hash (str): last commit hash of head.
+        remote_previous_commit_hash (str): previous commit of origin/master (origin/master~1)
+
+    """
+    if not os.path.exists(os.path.join(index_folder_path, f"{GCPConfig.INDEX_NAME}.json")):
+        # will happen only in init bucket run
+        print_warning(f"{GCPConfig.INDEX_NAME}.json not found in {GCPConfig.INDEX_NAME} folder")
+        return
+
+    skipping_build_task_message = "Skipping Upload Packs To Marketplace Storage Step."
+
+    try:
+        with open(os.path.join(index_folder_path, f"{GCPConfig.INDEX_NAME}.json")) as index_file:
+            index_json = json.load(index_file)
+
+        index_commit_hash = index_json.get('commit', remote_previous_commit_hash)
+        index_commit = content_repo.commit(index_commit_hash)
```

Comment:
What happens if the commit is not found? Are you handling this case?

Reply:
Yes, in line 516. `ValueError` is raised when no hash found.

Reply:
Would rather you specifically catch this error here, as ValueError is a generic error that something else may raise and then we would want to error out if something else threw this..